docs-chef-io/content/inspec/resources/docker_container.md

```diff
@@ -62,37 +62,37 @@ Alternatively, you can pass in the container id:
 
 The following examples show how to use this Chef InSpec resource.
 
-### id
+### ID
 
-The `id` property tests the container id:
+The `id` property tests the container ID.
 
     its('id') { should eq 'sha:71b5df59...442b' }
 
-### repo
+### Repository
```

Comment:
This should be `repo` and not `Repository` since Ruby identifiers are case sensitive and `repo` is the valid identifier here.